hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -562,15 +563,14 @@ public BlockMappingVO getBlockRowCount(Job job, CarbonTable table,
       if (CarbonProperties.getInstance()
           .isDistributedPruningEnabled(table.getDatabaseName(), table.getTableName())) {
         try {
-          List<InputSplit> extendedBlocklets = CarbonTableInputFormat.convertToCarbonInputSplit(
+          List<ExtendedBlocklet> extendedBlocklets =
               getDistributedSplit(table, null, partitions, filteredSegment,
-                  allSegments.getInvalidSegments(), toBeCleanedSegments));
-          for (InputSplit extendedBlocklet : extendedBlocklets) {
-            CarbonInputSplit blocklet = (CarbonInputSplit) extendedBlocklet;
+                  allSegments.getInvalidSegments(), toBeCleanedSegments);
```

Comment:
I think you supposed to pass true for count job

Reply:
done